api_docs/swagger/components/requestBodies/pre_activation/partner_order_payload.yaml

```diff
@@ -0,0 +1,28 @@
+type: object
+properties:
+  partner_order:
+    type: object
+    required: true
+    properties:
+      oem_token:
+        required: true
+        type: string
+        example: 987654
+      end_user_account_number:
```

Comment:
Can we accept UC site id's here, in addition to SF account numbers?
I ask because brand new SUSE customers will not have a SF account number.
Right now, we allow customers to enable OEM subs on a UC org, and at that point we create the account in SF and assign its SF account number.

Reply:
Do you think we should work only with the UC site IDs then? If this is data which the end user can always inform OEM partners, maybe it's a better option to consider.

Otherwise, do you think we should accept both UC site IDs and SF account numbers?

Reply:
From what I understand, most FJ customers are also SUSE customers, so they will have an SF account number.
But there will be some that are not SUSE customers yet, and will therefore not have an SF acct num.
Therefore, I think we could accept both.